src/effects/onSpreadValue/index.js

```diff
@@ -0,0 +1,11 @@
+import validate from '../validate';
+
+function onSpreadValue(selector = action => action.payload) {
+  return validate({
```

Comment:
This validation is not needed, since this effect doesn't have a single target.
This one can be achieved doing:
```
return (state, action) => mergeState(state, selector(action));
```

Reply:
ok! i'll wait until the other PR is merged